src/main/java/io/vertx/ext/eventbus/bridge/tcp/impl/TcpEventBusBridgeImpl.java

```diff
@@ -124,7 +125,8 @@ public TcpEventBusBridge listen(int port, Handler<AsyncResult<TcpEventBusBridge>
 
   private void handler(NetSocket socket) {
 
-    final Map<String, MessageConsumer<?>> registry = new HashMap<>();
+    final Map<String, MessageConsumer<?>> registry = new ConcurrentHashMap<>();
+    final Map<String, Message<JsonObject>> replies = new ConcurrentHashMap<>();
```

Comment:
The map is in the netsocket handler so it is per connection, replies should only be called once, so even if the message is sent to various clients they won't overwrite each other.